windows/src/engine/kmcomapi/kmcomapi.dpr

```diff
@@ -54,7 +54,6 @@ uses
   RegKeyboards in '..\..\global\delphi\general\RegKeyboards.pas',
   HotkeyUtils in '..\..\global\delphi\general\HotkeyUtils.pas',
   Glossary in '..\..\global\delphi\general\Glossary.pas',
-  olepro32 in 'util\olepro32.pas',
```

Comment:
This is being dropped as part of the x64 shift, I take it?

Reply:
Yes, olepro32 has been removed (long ago) and replaced with oleaut32. Win64 doesn't include olepro32 but Win32 had a stub redirect for backwards compat.